tests/test_polly/test_server.py

```diff
@@ -0,0 +1,16 @@
+from __future__ import unicode_literals
+
+import sure  # noqa
+
+import moto.server as server
+from moto import mock_polly
+
+'''
+Test the different server responses
+'''
+
+@mock_polly
+def test_polly_list():
+    backend = server.create_backend_app("polly")
+    test_client = backend.test_client()
+    # do test
```

Comment:
If we add the following will the tests pass?:
```
    res = test_client.get('/')
    res.status_code.should.equal(200)
```

Reply:
Ahh I completely forgot about that file. Will add that later today.